nuntium/tests/writeitinstances_test.py

```diff
@@ -372,3 +378,25 @@ def test_no_form_on_homepage_of_empty_instance(self):
         response = self.client.get(url)
 
         self.assertIn('there is no-one to write to', response.content)
+
+    def test_there_is_a_post_submission_url(self):
+        '''There is a post submission URL'''
+        message = self.writeitinstance1.message_set.get(id=1)
+        url = reverse('post_submission', kwargs={
+            'instance_slug': self.writeitinstance1.slug,
+            'slug': message.slug,
+            })
+        self.assertTrue(url)
```

Comment:
I don't think there's any point in having this assertion here - if reverse can't find a URL, it will raise ` NoReverseMatch`, so this won't be executed.